### ISSUE TITLE
Using pretrained model on Windows 10?

### ISSUE THREAD

**Issue by Jars-of-jam-Scheduler** [junyanz/pytorch-CycleGAN-and-pix2pix#1104]
Hello everyone,

Altough you have specified "MacOS and Linux" as prerequisites, do you know if it would be possible to make use of your pretrained model "summer2winter" on Windows (10)?

Thank you in advance.
Best regards,

**Comment by Jars-of-jam-Scheduler** [comment]
Yes it can be used.
